affinity/cpp-20/d0796r2.md

```diff
@@ -170,6 +170,27 @@ This feature could be easily scaled to heterogeneous and distributed systems, as
 
 In this paper we propose an interface for querying and representing the execution resources within a system, queurying the relative affinity metric between those execution resources, and then using those execution resources to allocate memory and execute work with affinity to the underlying hardware. The interface described in this paper builds on the existing initerface for executors and execution contexts defined in the executors proposal [[22]][p0443r4].
 
+### Terminology
+
+An **execution agent** executes work, typically implemented by a *callable*, 
+on an **execution resource** of a given **execution architecture**. 
+An **execution context** manages a set of execution agents on an 
+execution resource.
+An **executor** submits work to an execution context.
+More that one executor may submit work to an execution context.
+More than on execution context may manage execution agents
+on an execution resource.
+
+> [*Note:* The execution context terminology used here
+and in the Networking TS [[33]][networking-ts] deviate from the 
+traditional *context of execution* usage that refers
+to the state of a single executing callable; *e.g.*,
+program counter, registers, stack frame. *--end note*]
+
+The **concurrency** of an execution resource is the maximum number 
```

Comment:
I think we should add a note here based on our discussion on the email thread, that this is not a guarantee that this number of execution agents will always be capable of making forward progress.

Reply:
Done